api/app/Modules/Library/Http/Controllers/DraftLyricsController.php

```diff
@@ -27,8 +28,13 @@ public function __construct(DraftLyricsTransformer $transformer)
 
     public function index(ListDraftLyricsRequest $request): JsonResponse
     {
-        $draftLyrics = $request->track()->draftLyrics()->firstOrFail();
-        return $this->respondWithItem($draftLyrics);
+        if ($request->has('track_id')) {
+            $draftLyrics = $request->track()->draftLyrics()->firstOrFail();
+            return $this->respondWithItem($draftLyrics);
+        }
+        $draftLyrics = DraftLyrics::query()->orderBy("id")
```

Comment:
should the order by be `updated_at desc` or something so that its ordered by oldest to newest drafts?

Reply:
Done